runtime/build.gradle

```diff
@@ -101,3 +103,88 @@ task generateJsMath {
         }
     }
 }
+
+def clangFormatTargets = project.fileTree("src/main") {
+    include '**/*.c'
+    include '**/*.cpp'
+    include '**/*.h'
+    include '**/*.hpp'
+    include '**/*.m'
+    include '**/*.mm'
+
+    exclude 'cpp/utf8.h' // part of external library
+
+    // TODO: Reformat all these files.
```

Comment:
I think it's better to start gradually fixing these files instead of doing it in a single PR

Reply:
Put `// clang-format off` to every existing file's header.